src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAdminStatisticsService.java

```diff
@@ -86,17 +86,21 @@ public StatisticsResponse getAdminAnalysesStatistics(Integer timePeriod) {
 
 		if (IntStream.of(StatisticTimePeriod.DAILY.getDaily())
 				.anyMatch((x -> x == timePeriod))) {
-			return new StatisticsResponse(analysisSubmissionService.getAnalysesRanDaily(minimumCreatedDate));
+			return new StatisticsResponse(analysisSubmissionService.getAnalysesRanGrouped(minimumCreatedDate,
+					GroupByFormat.fromString("daily")));
```

Comment:
Use the enum values instead of `fromString`.  So `GroupByFormat.DAILY` instead here.

Reply:
This has been refactored in c5a00bcd1e78198ca591eaa499e2e494dd1cd5ca